performance/performance_tuning.rst

```diff
@@ -152,6 +152,14 @@ Differences in worker execution can be caused by tuning configuration difference
 
   Note that when citus.explain_all_tasks is enabled, EXPLAIN plans are retrieved sequentially, which may take a long time for EXPLAIN ANALYZE.
 
+For troubleshooting and detecting the bottlenecks, you can use this configuration variable:
+
+.. code-block:: postgresql
+
+  SET citus.explain_analyze_sort_method = 'execution-time';
+
+This will allow you to see all tasks sorted by execution time, in descending order, if ``citus.explain_all_tasks`` is on. i.e You can see the longest running task on top. If ``citus.explain_all_tasks`` is not enabled, then you will see only the longest running task. Please note that this functionality can only be used with ``EXPLAIN ANALYZE``, since ``EXPLAIN`` itself doesn't execute the queries, and therefore doesn't have any execution time. The default value of ``citus.explain_analyze_sort_method`` is ``execution-time``. Also, running ``SET citus.explain_analyze_sort_method = 'taskId';`` will allow you to get the output sorted by task ids.
```

Comment:
Nice GUC :)
Maybe after adding a reference section about the GUC in develop/api_guc.rst as @jonels-msft suggested, the last two sentences can be removed from here and be a part of the reference section.
We can also omit the quotation marks around citus.explain_all_tasks, EXPLAIN and EXPLAIN ANALYZE, as they are usually omitted in other paragraphs as well.